pkg/accounting/tally/tally_test.go

```diff
@@ -154,12 +145,74 @@ func TestCalculateAtRestData(t *testing.T) {
 		for _, actualTotalBytes := range actualNodeData {
 			assert.Equal(t, int64(actualTotalBytes), expectedTotalBytes)
 		}
+	})
+}
+
+func TestCalculateBucketAtRestData(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 6, UplinkCount: 1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		tallySvc := planet.Satellites[0].Accounting.Tally
+		uplink := planet.Uplinks[0]
+
+		// Setup: create 50KiB of data for the uplink to upload
+		expectedData := make([]byte, 50*memory.KiB)
+		_, err := rand.Read(expectedData)
+		require.NoError(t, err)
+
+		// Setup: get the expected size of the data that will be stored in pointer
+		uplinkConfig := uplink.GetConfig(planet.Satellites[0])
+		expectedTotalBytes, err := encryption.CalcEncryptedSize(int64(len(expectedData)), uplinkConfig.GetEncryptionScheme())
+		require.NoError(t, err)
+
+		// Setup: The data in this tally should match the pointer that the uplink.upload created
+		expectedBucketName1 := "testbucket1"
+		expectedTally1 := accounting.BucketTally{
+			BucketName:     []byte(expectedBucketName1),
+			Segments:       1,
+			RemoteSegments: 1,
+			Files:          1,
+			RemoteFiles:    1,
+			Bytes:          expectedTotalBytes,
+			RemoteBytes:    expectedTotalBytes,
+			MetadataSize:   112, // brittle, this is hardcoded since its too difficult to get this value progamatically
+		}
+
+		expectedBucketName2 := "testbucket2"
```

Comment:
nice, can we include a test to check if you upload the same data to a third bucket, those values match?